### ISSUE TITLE
[BUG]: Content doesn't load without internet connection

### ISSUE THREAD

**Issue by cameronj86** [ankidroid/Anki-Android#14018]
### Checked for duplicates?

- [X] This issue is not a duplicate

### What are the steps to reproduce this bug?

1. Have device in in airplane mode
2. Open app for the first time of the day

### Expected behaviour

App should either notify of inability to sync and load flashcards, or just load flashcards. 

### Actual behaviour

App attempts to load indefinitely w/o populating the offline flashcards

[Video example](https://fosstodon.org/@macallik/110584955973636657)

### Debug info

```text
AnkiDroid Version = 2.16.0beta2

Android Version = 13

ProductFlavor = full

Manufacturer = Xiaomi

Model = 23043RP34C

Hardware = qcom

Webview User Agent = Mozilla/5.0 (Linux; Android 13; 23043RP34C Build/TKQ1.221114.001; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/114.0.5735.131 Safari/537.36

ACRA UUID = 6716e10b-1721-4a5e-b7fc-5033264ba3ad

New schema = false

Scheduler = std2

Crash Reports Enabled = true

DatabaseV2 Enabled = true
```

### (Optional) Anything else you want to share?

1. Although it happens basically every time, instantly exiting -> reopening app resolves the issue
2. I am using a tablet so my offline usage is probably a lot greater than most users on their phones

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by doozan** [comment]
Possibly backend related? Does it still hang if you switch to the new backend (Preferences, Advanced, near the bottom)? The new backend has its own bugs with offline mode (#12821) but maybe an error message is better than a hang.

**Comment by BrayanDSO** [comment]
Should be fixed in 2.17
